apps/demo-react/components/contractTesting/Stake.tsx

```diff
@@ -161,7 +161,7 @@ const StakeForm = ({
     void providerWeb3
       ?.getBalance(account || '')
       .then((data) => setWalletBalance(formatBalance(data, 5)), console.log);
```

Comment:
can you replace `console.log` with `console.warn` here? maybe in other PR

Reply:
This is not my code, and it is touched in this PR only to fix eslint errors.
@TonyCoopeR-62 check this @Jabher comments, can you take them to the next demo app related PR?